lib/gomod-parser.ts

```diff
@@ -173,3 +177,61 @@ export function toSnykVersion(v: ModuleVersion): string {
     return (v as ModuleExactVersion).exactVersion;
   }
 }
+
+export async function buildDepTreeFromImports(path: string = '.') {
+  const depTree: DepTree = {
+    name: 'tempName', // TODO: How do get name of the module?
```

Comment:
by using https://github.com/snyk/snyk-go-parser ?

Also, module or package?

Reply:
Module in this case, this is the top level name, so user will be testing their module.